connectors/agentic-ai/src/main/java/io/camunda/connector/agenticai/adhoctoolsschema/resolver/CamundaClientAdHocToolsSchemaResolver.java

```diff
@@ -66,29 +78,46 @@ public AdHocToolsSchemaResponse resolveSchema(
   }
 
   private AdHocToolDefinition mapActivityToToolDefinition(FlowNode element) {
-    final var documentation = getDocumentation(element);
-
-    if (!documentation.isBlank() && documentation.trim().startsWith("{")) {
```

Comment:
No JSON parsing on the documentation anymore - instead we extract `fromAi` function calls from input & output mappings.